packages/components/src/components/common/y-axis/as-y-axis.scss

```diff
@@ -0,0 +1,17 @@
+.y-axis {
+  .tick {
+    text {
+      width: 30px;
+      fill: var(--widget-axis-text-color, rgb(44, 44, 44));
+      white-space: pre;
+    }
+
+    line {
+      stroke: var(--widget-axis-line-color, rgba(46, 60, 67, 0.16));
```

Comment:
I like having this as a var, but the color is not consistent with the histogram. I would add it there as well, and use the same fallback on both.

Either here or on a new "Sync histogram" PR